types/mongodb/index.d.ts

```diff
@@ -2378,12 +2429,82 @@ export class ChangeStream extends Readable {
 
 export class ResumeToken {}
 
+export type ChangeEventTypes = 'insert' | 'delete' | 'replace' | 'update' | 'drop' | 'rename' | 'dropDatabase' | 'invalidate';
+export interface ChangeEventBase<TSchema extends object = {_id: ObjectId}> {
+    _id: ResumeToken;
+    /**
+     * We leave this off the base type so that we can differentiate
+     * by checking its value and get intelligent types on the other fields
+     */
+    // operationType: ChangeEventTypes;
+    ns: {
+        db: string;
+        coll: string;
+    };
+    clusterTime: Timestamp;
+    txnNumber?: number;
+    lsid?: {
+        "id": any;
+        "uid": any;
+    };
+}
+export interface ChangeEventCR<TSchema extends object = {_id: ObjectId}> extends ChangeEventBase<TSchema> {
```

Comment:
Same here with the `extends` and `=` type, and a few places below as well

Reply:
done